DirectProgramming/DPC++FPGA/Tutorials/Features/floating_point_pragmas/README.md

```diff
@@ -0,0 +1,190 @@
+    cmake -G "NMake Makefiles" ..
+   ```
+   Alternatively, to compile for the Intel® FPGA PAC D5005 (with Intel Stratix® 10 SX), run `cmake` using the command:
+
+   ```
+   cmake -G "NMake Makefiles" .. -DFPGA_BOARD=intel_s10sx_pac:pac_s10
+   ```
+
+2. Compile the design through the generated `Makefile`. The following build targets are provided, matching the recommended development flow:
+
+   * Compile for emulation (fast compile time, targets emulated FPGA device):
+     ```
+     nmake fpga_emu
+     ```
+   * Generate the optimization report:
+     ```
+     nmake report
+     ```
+   * An FPGA hardware target is not provided on Windows*.
+
+*Note:* The Intel® PAC with Intel Arria® 10 GX FPGA and Intel® FPGA PAC D5005 (with Intel Stratix® 10 SX) do not yet support Windows*. Compiling to FPGA hardware on Windows* requires a third-party or custom Board Support Package (BSP) with Windows* support.
+
+### In Third-Party Integrated Development Environments (IDEs)
+
+You can compile and run this tutorial in the Eclipse* IDE (in Linux*) and the Visual Studio* IDE (in Windows*). For instructions, refer to the following link: [Intel® oneAPI DPC++ FPGA Workflows on Third-Party IDEs](https://software.intel.com/en-us/articles/intel-oneapi-dpcpp-fpga-workflow-on-ide)
+
+## Examining the Reports
+
+Locate the pair of `report.html` files in either:
+
+* **Report-only compile**:  `fp_pragmas_report.prj`
+* **FPGA hardware compile**: `fp_pragmas.prj`
+
+Open the reports in Google Chrome*, Mozilla Firefox*, Microsoft Edge*, or Microsoft Internet Explorer*.
+
+On the main report page, scroll down to the section titled "Estimated Resource Usage". Each kernel name represents the pragma usage. e.g., `ContractFastKernel` sets the `fp contract(fast|off)` pragma to `fast` mode. You can verify that the number of ALMs used for kernels with `fp contract(fast)` or `fp reassociate(on)` is fewer than the kernels with `fp contract(off)` or `fp reassociate(off)` respectively.
+
+In the "Loop Analysis" section under the tab titled "Throughput Analysis", there is a panel on the left containing all the kernels. the loop information of each kernel can be seen by expanding them. You can verify that the latencies for kernels with `fp contract(fast)` or `fp reassociate(on)` are smaller than those of the kernels with `fp contract(off)` or `fp reassociate(off)` respectively.
+
+## Running the Sample
+
+1. Run the sample on the FPGA emulator (the kernel executes on the CPU):
+
+   ```bash
+   ./fp_pragmas.fpga_emu    # Linux
+   fp_pragmas.fpga_emu.exe  # Windows
+   ```
+
+2. Run the sample on the FPGA device
+
+   ```bash
+   ./fp_pragmas.fpga        # Linux
+
+### Example of Output
+
+```txt
+PASSED: The results are correct
+```
+
+### Discussion of Results
+
+You can see the reduction of resource use in kernel ContractFastKernel comparing with ContractOffKernel. You will also note the decrease in resource use in kernel ReassociateOnKernel comparing with ReassociateOffKernel.
```

Comment:
The source code and README.md is modified to show the benefit of turning on optimizations instead of showing the drawbacks of turning off optimizations.

Reply:
Unresolving this so I remember to take a look.  This doesn't sound like the correct approach to me, since the optimization is on by default.  We don't have to show the users the benefits of using the default, we have to explain to them why and when they would NOT use the default.  I'll review your readme changes and 're-resolve' this conversation if it looks good . . .

Reply:
OK, I've reviewed it and this is not the approach I think we should take.  Please reach out to me and we'll discuss offline.

Reply:
Modified README.md and source code to reflect the change